onnxruntime/test/providers/cuda/cuda_mempool_arena_test.cc

```diff
@@ -25,26 +25,33 @@ namespace test {
 static bool IsCudaMemPoolSupported() {
   int ort_cuda_rt_version = 0;
   cudaError_t cuda_status = cudaRuntimeGetVersion(&ort_cuda_rt_version);
-  bool version_supported = (cuda_status == cudaSuccess && ort_cuda_rt_version >= 11020);
-  if (!version_supported) {
+  if (cuda_status != cudaSuccess) {
```

Comment:
Nit: I guess we can just re-use IsCudaVersionSupported() from the memory pool class ?